rust/kernel/src/sync.rs

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: GPL-2.0
+
+use core::cell::UnsafeCell;
+use core::ops::{Deref, DerefMut};
+use core::fmt;
+
+use crate::bindings;
+
+/*#[cfg(CONFIG_DEBUG_SPINLOCK)]
```

Comment:
If you move the `/*` and `*/` to their own line, then a future diff is smaller.

Reply:
removed completly

Reply:
I'm confused, why are these commented out?

Reply:
Removed it after implementing the DEBUG_ALOC stuff